.github/workflows/pull-request-check-rust-api.yaml

```diff
@@ -62,14 +62,14 @@ jobs:
       # by the other jobs already present in `pull-request-checks.yaml`.
       - name: Run Rust API tests
         run: |
-          VERSION=$(cat src/config.inc | python3 -c "import sys,re;line = [line for line in sys.stdin if re.search('CBMC_VERSION = (\d+\.\d+\.\d+)', line)];sys.stdout.write(re.search('CBMC_VERSION = (\d+\.\d+\.\d+)', line[0]).group(1))")
+          VERSION=$(cat src/config.inc | python3 -c "import sys,re;line = [line for line in sys.stdin if re.search(r'CBMC_VERSION = (\d+\.\d+\.\d+)', line)];sys.stdout.write(re.search(r'CBMC_VERSION = (\d+\.\d+\.\d+)', line[0]).group(1))")
           cd src/libcprover-rust;\
           cargo clean;\
           CBMC_INCLUDE_DIR=../../${{env.default_include_dir}} CBMC_LIB_DIR=../../${{env.default_build_dir}}/lib CBMC_VERSION=$VERSION cargo test -- --test-threads=1
 
 
-  check-macos-12-cmake-clang-rust:
-    runs-on: macos-12
+  check-macos-13-cmake-clang-rust:
+    runs-on: macos-13
```

Comment:
This change isn't actually necessary, is it?

Reply:
It's not needed, but to be honest I'm inclined to keep, just so we expand our testing targets a bit (On personal machines I'm on 14, as probably does @esteffin because of internal IT policy), and we're only testing 11 and 12 on the `pull-request-checks.yaml` file.

The only downside I can think of for not making this change is potential cache contention (as in, the size the cache requires, because I believe we're already quite close to our allowed limit, if not over it already). But I think even with this downside the upside is probably worth it.

Any reason *not* to keep this one in particular?

Reply:
No big deal, this PR is a mixed bag of CI changes already, so I think you are right that we should keep this.